config/v2_3_experimental/types/schema.go

```diff
@@ -189,7 +189,7 @@ type RaidOption string
 type SSHAuthorizedKey string
 
 type Security struct {
-	TLS TLS `json:"tls,omitempty"`
+	TLS `json:"tls,omitempty"`
```

Comment:
What's going on here?

Reply:
Not sure. This is just what `schematyper` gave me.